### ISSUE TITLE
Create Mockup for Password Validation Requirements

### ISSUE THREAD

**Issue by b-f-chan** [OHCRN/platform#692]
Create mockup for password validation requirements

- Minimum length 12 characters
- Mix of upper and lowercase characters

**Comment by b-f-chan** [comment]
Yes, we are moving forward with the following, as confirmed with @joneubank 

- Remove password confirmation box
- Instead adding granular error check messages

**Comment by patrickdsantos** [comment]
<img width="922" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMUcxQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--bbabb15f13d4be7019c92d229a39c44c8f51a7c0/step2.png" alt="step2.png" />
